website/sidebars.json

```diff
@@ -4,9 +4,17 @@
       "what-why",
       "getting-started",
       "configuration",
+      "esy-configuration",
       "environment",
-      "commands"
+      "commands",
+      "concepts"
     ],
-    "Advanced": ["how-it-works", "development"]
+    "Advanced": [
+      "release"
+    ],
+    "Development Documentation": [
```

Comment:
Long sentences look weird on section titles:
![image](https://user-images.githubusercontent.com/9691/34284317-f749b15e-e70b-11e7-9df4-c7c25bd461eb.png)

Reply:
Changed to `Internals`